server/controllers/refer/referralsController.test.ts

```diff
@@ -145,4 +146,42 @@ describe('ReferralsController', () => {
       })
     })
   })
+
+  describe('checkAnswers', () => {
+    it('renders the referral check answers page', async () => {
+      const organisation = organisationFactory.build({ id: courseOffering.organisationId })
+      organisationService.getOrganisation.mockResolvedValue(organisation)
+
+      const person = personFactory.build()
+      personService.getPerson.mockResolvedValue(person)
+
+      const referral = referralFactory.build({ offeringId: courseOffering.id, prisonNumber: person.prisonNumber })
+      referralService.getReferral.mockResolvedValue(referral)
+
+      request.params.referralId = referral.id
+
+      const coursePresenter = createMock<CoursePresenter>({
+        audiences: courseAudienceFactory.buildList(1),
+        nameAndAlternateName: `${course.name} (${course.alternateName})`,
+      })
```

Comment:
I know we're doing this elsewhere, but I wonder what the use is of setting these values on the mocked `coursePresenter` when we aren't asserting on them. Probably another conversation for later!